dali/operators/reader/loader/video_loader.h

```diff
@@ -242,6 +242,9 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
                                    codecpar(stream)->height, codecpar(stream)->width});
       }
     }
+    DALI_ENFORCE(frame_starts_.size() != 0, "There are no valid sequences in the provided "
+                 "dataset, check the length of the available videos and the requested sequence "
+                 "length.");
```

Comment:
Is that the actual fix?

Reply:
Yes, instead of segv when trying to access an element from the empty `frame_starts_` array, we just propagate a proper error to python.